nucypher/cli/painting.py

```diff
@@ -301,22 +303,61 @@ def paint_staged_stake_division(emitter,
                        division_message=division_message)
 
 
-def paint_contract_deployment(emitter, contract_name: str, contract_address: str, receipts: dict):
+def paint_receipt_summary(emitter, receipt, chain_name, transaction_type=None):
+    tx_hash = receipt['transactionHash'].hex()
+    emitter.echo("OK", color='green', nl=False, bold=True)
+    if transaction_type:
+        emitter.echo(f" | {transaction_type} | {tx_hash}", color='yellow', nl=False)
+    else:
+        emitter.echo(f" | {tx_hash}", color='yellow', nl=False)
+    emitter.echo(f" ({receipt['gasUsed']} gas)")
+    emitter.echo(f"Block #{receipt['blockNumber']} | {receipt['blockHash'].hex()}")
+    try:
+        url = etherscan_url(item=tx_hash, network=chain_name)
+    except ValueError as e:
+        emitter.log.info("Failed Etherscan URL construction: " + str(e))
+    else:
+        emitter.echo(f" See {url}\n")
+
+
+def paint_contract_deployment(emitter,
+                              contract_name: str,
+                              contract_address: str,
+                              receipts: dict,
+                              chain_name: str = None,
+                              open_in_browser: bool = False):
 
     # TODO: switch to using an explicit emitter
 
+    is_token_contract = contract_name == NUCYPHER_TOKEN_CONTRACT_NAME
+
     # Paint heading
     heading = f'\r{" "*80}\n{contract_name} ({contract_address})'
     emitter.echo(heading, bold=True)
     emitter.echo('*' * (42 + 3 + len(contract_name)))
+    try:
+        url = etherscan_url(item=contract_address, network=chain_name, is_token=is_token_contract)
+    except ValueError as e:
+        emitter.log.info("Failed Etherscan URL construction: " + str(e))
+    else:
+        emitter.echo(f" See {url}\n")
 
     # Paint Transactions
     for tx_name, receipt in receipts.items():
-        emitter.echo("OK", color='green', nl=False, bold=True)
-        emitter.echo(" | {}".format(tx_name), color='yellow', nl=False)
-        emitter.echo(" | {}".format(receipt['transactionHash'].hex()), color='yellow', nl=False)
-        emitter.echo(" ({} gas)".format(receipt['cumulativeGasUsed']))
-        emitter.echo("Block #{} | {}\n".format(receipt['blockNumber'], receipt['blockHash'].hex()))
+        paint_receipt_summary(emitter=emitter,
+                              receipt=receipt,
+                              chain_name=chain_name,
+                              transaction_type=tx_name)
+
+    if open_in_browser:
+        try:
+            url = etherscan_url(item=contract_address,
+                                network=chain_name,
+                                is_token=is_token_contract)
+        except ValueError as e:
+            emitter.log.info("Failed Etherscan URL construction: " + str(e))
+        else:
```

Comment:
I would try to import `webbrowser` right here (in case we deploy from a commandline-only remote machine)

Reply:
I tried in my remote machine, and there's no problem at all. It simply outputs `False`
```python
>>> webbrowser.open_new_tab("https://github.com/")
False
```

Reply:
Huh, didn't realize it's a built-in module. I think, all good in that case